### PR TITLE
Bump MLServer version to 1.2.0

### PR DESCRIPTION
**Pull request by adriangonz**
**What this PR does / why we need it**:
Bump MLServer to 1.2.0

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:
We will first run the tests with MLServer 1.2.0.RC1. If everything goes well, we'll cut the final `1.2.0` release and update this PR.

### CODE REVIEW

**Comment by seldondev** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **adriangonz**
You can assign the PR to them by writing `/assign @adriangonz` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://jenkins-x.io/v3/develop/reference/chatops/?repo=SeldonIO%2Fseldon-core).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/SeldonIO/seldon-core/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by seldondev** [comment]
@adriangonz: The following test **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
integration | 5c2616a16f4f1a50ca97c99411600d2d05eb7947 | [link](http://dashboard-jx.34.121.152.94.nip.io/namespaces/jx/pipelineruns/seldon-core-pr-4448-integration-t2lwv) | `/test integration`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [jenkins-x/lighthouse](https://github.com/jenkins-x/lighthouse/issues/new?title=Command%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by seldondev** [comment]
@adriangonz: The following tests **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
integration | 5c2616a16f4f1a50ca97c99411600d2d05eb7947 | [link](http://dashboard-jx.34.121.152.94.nip.io/namespaces/jx/pipelineruns/seldon-core-pr-4448-integration-t2lwv) | `/test integration`
notebooks | 5c2616a16f4f1a50ca97c99411600d2d05eb7947 | [link](http://dashboard-jx.34.121.152.94.nip.io/namespaces/jx/pipelineruns/io-seldon-core-pr-4448-notebooks-p5fm5) | `/test notebooks`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [jenkins-x/lighthouse](https://github.com/jenkins-x/lighthouse/issues/new?title=Command%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by seldondev** [comment]
@adriangonz: The following tests **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
notebooks | 5c2616a16f4f1a50ca97c99411600d2d05eb7947 | [link](http://dashboard-jx.34.121.152.94.nip.io/namespaces/jx/pipelineruns/io-seldon-core-pr-4448-notebooks-p5fm5) | `/test notebooks`
integration | b2b9f2c15c62a4d77320d2f552bb44ef8cdb3b52 | [link](http://dashboard-jx.34.121.152.94.nip.io/namespaces/jx/pipelineruns/seldon-core-pr-4448-integration-mvgn2) | `/test integration`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [jenkins-x/lighthouse](https://github.com/jenkins-x/lighthouse/issues/new?title=Command%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by seldondev** [comment]
@adriangonz: The following tests **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
notebooks | 5c2616a16f4f1a50ca97c99411600d2d05eb7947 | [link](http://dashboard-jx.34.121.152.94.nip.io/namespaces/jx/pipelineruns/io-seldon-core-pr-4448-notebooks-p5fm5) | `/test notebooks`
integration | 6b185bb9023ba0c4c0b3eff00fe8014a4827f1de | [link](http://dashboard-jx.34.121.152.94.nip.io/namespaces/jx/pipelineruns/seldon-core-pr-4448-integration-ttz8x) | `/test integration`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [jenkins-x/lighthouse](https://github.com/jenkins-x/lighthouse/issues/new?title=Command%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by seldondev** [comment]
@adriangonz: The following test **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
integration | 6b185bb9023ba0c4c0b3eff00fe8014a4827f1de | [link](http://dashboard-jx.34.121.152.94.nip.io/namespaces/jx/pipelineruns/seldon-core-pr-4448-integration-b8tnk) | `/test integration`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [jenkins-x/lighthouse](https://github.com/jenkins-x/lighthouse/issues/new?title=Command%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by seldondev** [comment]
@adriangonz: The following test **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
integration | 6b185bb9023ba0c4c0b3eff00fe8014a4827f1de | [link](http://dashboard-jx.34.121.152.94.nip.io/namespaces/jx/pipelineruns/seldon-core-pr-4448-integration-zsbnj) | `/test integration`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [jenkins-x/lighthouse](https://github.com/jenkins-x/lighthouse/issues/new?title=Command%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by adriangonz** [comment]
Notebooks tests seem to pass, but there's still something going on with Jaeger on the integration tests. I'll try to replicate locally. 

image

**Comment by seldondev** [comment]
@adriangonz: The following test **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
integration | 87889a35bb6cbd13401b3d28119d7ac5336027ac | [link](http://dashboard-jx.34.121.152.94.nip.io/namespaces/jx/pipelineruns/seldon-core-pr-4448-integration-dgd6g) | `/test integration`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [jenkins-x/lighthouse](https://github.com/jenkins-x/lighthouse/issues/new?title=Command%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by RafalSkolasinski** [comment]
> Notebooks tests seem to pass, but there's still something going on with Jaeger on the integration tests. I'll try to replicate locally.

integration passed without issues here  https://github.com/SeldonIO/seldon-core/pull/4458 so maybe just flakiness?

**Comment by adriangonz** [comment]
@RafalSkolasinski it was mainly that Jaeger was taking a bit longer to get installed. I've bumped up the timeout to 60s (from the default 30s) and that seems to be more reliable.

Besides that, one of the Triton tests also timed out while pulling the Triton image. I've re-ran that one locally and seemed to pass though, so hopefully re-triggering the integration tests should work now :+1:

**Comment by seldondev** [comment]
@adriangonz: The following test **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
integration | 87889a35bb6cbd13401b3d28119d7ac5336027ac | [link](http://dashboard-jx.34.121.152.94.nip.io/namespaces/jx/pipelineruns/seldon-core-pr-4448-integration-77kj2) | `/test integration`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [jenkins-x/lighthouse](https://github.com/jenkins-x/lighthouse/issues/new?title=Command%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by seldondev** [comment]
@adriangonz: The following test **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
integration | 54561f1ae9015d08f6ae021a4a02e2b6fd87b215 | [link](http://dashboard-jx.34.121.152.94.nip.io/namespaces/jx/pipelineruns/seldon-core-pr-4448-integration-r762d) | `/test integration`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [jenkins-x/lighthouse](https://github.com/jenkins-x/lighthouse/issues/new?title=Command%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by seldondev** [comment]
@adriangonz: The following tests **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
integration | 54561f1ae9015d08f6ae021a4a02e2b6fd87b215 | [link](http://dashboard-jx.34.121.152.94.nip.io/namespaces/jx/pipelineruns/seldon-core-pr-4448-integration-r762d) | `/test integration`
notebooks | 54561f1ae9015d08f6ae021a4a02e2b6fd87b215 | [link](http://dashboard-jx.34.121.152.94.nip.io/namespaces/jx/pipelineruns/io-seldon-core-pr-4448-notebooks-vcftl) | `/test notebooks`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [jenkins-x/lighthouse](https://github.com/jenkins-x/lighthouse/issues/new?title=Command%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by seldondev** [comment]
@adriangonz: The following tests **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
notebooks | 54561f1ae9015d08f6ae021a4a02e2b6fd87b215 | [link](http://dashboard-jx.34.121.152.94.nip.io/namespaces/jx/pipelineruns/io-seldon-core-pr-4448-notebooks-vcftl) | `/test notebooks`
integration | 66014fe98f801b60e4b9da34ef239ce7e102128c | [link](http://dashboard-jx.34.121.152.94.nip.io/namespaces/jx/pipelineruns/seldon-core-pr-4448-integration-hx6v9) | `/test integration`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [jenkins-x/lighthouse](https://github.com/jenkins-x/lighthouse/issues/new?title=Command%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by seldondev** [comment]
@adriangonz: The following test **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
integration | c949a5827acc8fd59a6e4f85c08e51fbb7bc52ad | [link](http://dashboard-jx.34.121.152.94.nip.io/namespaces/jx/pipelineruns/seldon-core-pr-4448-integration-9p752) | `/test integration`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [jenkins-x/lighthouse](https://github.com/jenkins-x/lighthouse/issues/new?title=Command%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by adriangonz** [comment]
Yay! Integration passed!

image

**Comment by RafalSkolasinski** [comment]
image

File: helm-charts/seldon-core-operator/README.md
Comment (RafalSkolasinski) [inline helm-charts/seldon-core-operator/README.md:16]:
```suggestion
Once that's done, you should then be able to deploy the chart as:
```

File: testing/scripts/test_alibi_explain.py
Comment (RafalSkolasinski) [inline testing/scripts/test_alibi_explain.py:58]: nit: `wait_for_deployment(name, namespace)` equivalents are called inside `wait_for_rollout` so second call is redundant.

**Review by RafalSkolasinski — APPROVED** [review]

**Comment by seldondev** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *[RafalSkolasinski](https://github.com/SeldonIO/seldon-core/pull/4448#pullrequestreview-1193476956 "Approved")*

The full list of commands accepted by this bot can be found [here](https://jenkins-x.io/v3/develop/reference/chatops/?repo=SeldonIO%2Fseldon-core).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/SeldonIO/seldon-core/blob/master/OWNERS)~~ [RafalSkolasinski]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by seldondev** [comment]
Failed to merge this PR due to:
>failed merging [4448]: [Method Not Allowed]

**Comment by seldondev** [comment]
Failed to merge this PR due to:
>failed merging [4448]: [Method Not Allowed]

**Comment by seldondev** [comment]
Failed to merge this PR due to:
>failed merging [4448]: [Method Not Allowed]

**Comment by seldondev** [comment]
Failed to merge this PR due to:
>failed merging [4448]: [Method Not Allowed]

**Comment by seldondev** [comment]
Failed to merge this PR due to:
>failed merging [4448]: [Method Not Allowed]

**Comment by seldondev** [comment]
Failed to merge this PR due to:
>failed merging [4448]: [Method Not Allowed]

**Comment by seldondev** [comment]
Failed to merge this PR due to:
>failed merging [4448]: [Method Not Allowed]

**Comment by seldondev** [comment]
Failed to merge this PR due to:
>failed merging [4448]: [Method Not Allowed]

**Comment by seldondev** [comment]
Failed to merge this PR due to:
>failed merging [4448]: [Method Not Allowed]

**Comment by seldondev** [comment]
Failed to merge this PR due to:
>failed merging [4448]: [Method Not Allowed]

**Comment by seldondev** [comment]
Failed to merge this PR due to:
>failed merging [4448]: [Method Not Allowed]

**Pull request merged**
